supplier_app/forms.py

```diff
@@ -79,11 +107,38 @@ class Meta:
             'razon_social': forms.TextInput(attrs={'class': 'form-control'}),
             'cuit': forms.TextInput(attrs={'class': 'form-control'}),
             'forma_de_pago': forms.TextInput(attrs={'class': 'form-control'}),
-            'justificacion': forms.Textarea(attrs={'class': 'form-control', 'rows': '3'})
+            'justificacion': forms.Textarea(attrs={
+                'class': 'form-control',
+                'rows': '3'
+            }),
+            'inscripcion_AFIP_file': forms.FileInput(attrs={
+                'accept': 'application/pdf',
+                'class': 'form-control'
+            }),
+            'retencion_file': forms.FileInput(attrs={
+                'accept': 'application/pdf',
+                'class': 'form-control'
+            }),
         }
 
+    def is_valid(self):
+        valid = super().is_valid()
+        if not valid:
+            return valid
```

Comment:
I'm not sure what's doing this section, could you explain me?

Reply:
I have to execute "super().is_valid()" because BaseForm class execute the full_clean method that is necessary to set the attribute cleaned_data